Update Generator_InjectedPromptCharmoniaFwdy_TriggerGap_PbPb5TeV.ini

sgaretti opened the pull request.

lucamicheletti93 commented:
Daer @sawenzel ! Sorry for the inconvenience! Probably there has been a misunderstanding when opening the PR. We are checking it!
Luca

sgaretti reviewed (commented):
Now pythia8_hi.cfg will be used as Pythia8 generator

sgaretti commented:
Dear @sawenzel,

I realized that some of the changes I initially made were not intended for this PR, so I have removed them. 
My apologies for the confusion!
Now, everything should be correct, and the only modified file is Generator_InjectedPromptCharmoniaFwdy_TriggerGap_PbPb5TeV.ini.
I also added a comment, which I had forgotten to include earlier.

Please let me know if there are any further issues; I can close this PR and open a new one if necessary.
Sara

sgaretti commented:
Dear @sawenzel,
Could you please have a look at the PR now?
Thank you in advance,
Sara
